Detectors/MUON/MCH/Evaluation/README.md

```diff
@@ -17,3 +17,40 @@ $ o2-mch-tracks-reader-workflow --hbfutils-config o2_tfidinfo.root --infile mcht
 $ root
 root[0] o2::mch::eval::drawAll("compare.root")
 ```
+# MCH Cluster Maps
+
+The general purpose is to track "unexpected" detector issues not well reproduced with MC simulations. These problems generate non-negligible bias in Acc*Eff corrections resulting in large tracking systematic uncertainties.
+
+During the data reconstruction, the status of the detector is calculated with the CCDB which is used to discard most of the detector issues.
+This status map is built with information based on pedestals, high voltage tension, occupancy etc.
+
+Nevertheless, some detector issues (e.g. a cable swapping) are not well detected online and consequently not properly reproduced by the CCBD.
+The main objective of this code is to spot these issues not included in the status map.
+
+Input files used:
+
+DATA_QC.root
+MC_QC.root
+o2sim_geometry-aligned.root
+
+Src file:
+map_mch.cxx
+
+COMPILATION (commands):
+
+cd alice ==> alienv enter O2/latest
+cd sw/BUILD/O2-latest/O2 ==> cmake --build .
+
+EXECUTION (command):
+
+cd InputFiles ==> o2-mch-map_mch --green --normperarea --rootfileleft DATA_QC.root --rootfileright MC_QC.root
+
+HELP MESSAGE (command):
+
+o2-mch-map_mch --help
+
+OPEN OUTPUT FILES:
+
+cd outputfile ==> open CHAMBERS-1-NB.html CHAMBERS-2-NB.html CHAMBERS-3-NB.html CHAMBERS-4-NB.html CHAMBERS-5-NB.html CHAMBERS-6-NB.html CHAMBERS-7-NB.html CHAMBERS-8-NB.html CHAMBERS-9-NB.html CHAMBERS-10-NB.html
```

Comment:
I believe the `open` command is a Mac-only thing, so not reproducible e.g. on a Linux machine. Either change it to plain English (open in your preferred browser the CHAMBERS*.html files generated) or provide the equivalent Linux command...